README.md

```diff
@@ -2,19 +2,104 @@
 
 > Test client for Xo-Server
 
-## Usage
-
-xo-server needs to be installed on `localhost:9000`.
-You should also have a `admin@admin.net` account with password `admin`.
+You should have a `admin@admin.net` account with password `admin`.
 
-There should be a Xen Server available at `192.168.1.3`with the credentials `root` and password `qwerty`.
+## Adding a test
 
-Warning ! This client will create and delete a lot of items (users, groups, servers ...) to test features
+### Organization
 
 ```
-> npm ci
-> npm test
+src
+├─ user
+|   ├─ __snapshots__
+|   |     └─ index.spec.js.snap 
+|   └─ index.spec.js
+├─ job
+¦   └─ index.spec.js
+¦
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
+    
+Let's take an example of `index.spec.js` from `user/`:  
```

Comment:
The developer can go to the file to see an example. But we can explain how it's organised. Like saying that the tests can describes a method or a scenario.

Reply:
Are you sure that the organisation of test files will be understandable without this example ? I think, it's easier to understand by this example and its comments than by a sentence. I can add an explanation saying that `the tests can describes a method or a scenario`, if you want. What do you think ?

Reply:
I means the sentence. Instead of saying it's an extract of the user tests. We can say that the tests can describe a method or a scenario.

Reply:
Ok :+1: